Add wordbreak character support [RTL-8689] (#91)
NextCapital/copy-service: README.md

    "save": "<strong>Save</strong>",
    "cancel": "<q>Cancel</q>",
    "rollover": "^{You may rollover}[rollover]",
    "implement": "^{You are going to implement}[implement][true]",

>     "wordbreak": "Add\b this"

Reviewer: ```suggestion
    "wordbreak": "Step\b 3 of 10"
```

- With the `PlainTextEvaluator`, `account.rollover` with substitutions `{ rollover: (copy) => copy + 'your external IRA' }` will resolve to `'You may rollover'` if `allowFormatting` is false, and `'You may rollover your external IRA'` if it is `true`.
- With the `ReactEvaluator`, `account.rollover` will resolve to `You may rollover your external IRA`.
- With the `HtmlEvaluator`, `account.implement` with substitutions `{ implement: (copy, addMore) => copy + 'your NextCapital managed account' }` will resolve to `'You are going to implement your NextCapital managed account'`.

> - With the `HtmlEvaluator` and `ReactEvaluator`, `account.wordbreak` will resolve to `Add<wbr/> this`. Note that `\b` in our syntax is used to represent a custom wordbreak character instead of the traditional regex match character.

Reviewer: ```suggestion
- With the `HtmlEvaluator` and `ReactEvaluator`, `account.wordbreak` will resolve to `Step<wbr/> 3 of 10`. Note: `\b` in custom language syntax to this project!
```
